src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionContainerBase.cs

```diff
@@ -515,22 +516,21 @@ private async Task WriteToScopedOrRandomAvailableConnection(ServiceMessage servi
             }
             else
             {
-                await WriteWithRetry(serviceMessage, null, currentConnections);
+                var connectionUsed = await WriteWithRetry(serviceMessage, _inUseConnection, currentConnections, IterateConnectionsInFixedOrder);
```

Comment:
so it changes the current behavior that the traffic distributes to random connections for proxy mode. is using random connections enough?